modules/virt-about-ksm.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-activating-ksm.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-about-ksm_{context}"]
+= About using {VirtProductName} to activate KSM
+
+You can configure {VirtProductName} to activate KSM when nodes experience memory overload.
+
+[id="virt-ksm-configuration-methods"]
+== Configuration methods
+
+* To enable or disable this feature for all nodes, you can use the {product-title} web console or edit the `spec.configuration.ksmConfiguration` stanza of the `HyperConverged` custom resource (CR). Deleting the `ksmConfiguration` stanza from the `HyperConverged` CR disables the feature.
```

Comment:
```suggestion
* To enable or disable this feature for all nodes, you can use the {product-title} web console or edit the `spec.configuration.ksmConfiguration` stanza of the `HyperConverged` custom resource (CR). Enable the feature by configuring settings in the `ksmConfiguration` stanza. Disable the feature by deleting the stanza from the `HyperConverged` CR.
```
- I was just wondering why there's a specific sentence about disabling the feature, but not a specific sentence about enabling it first. The paragraph begins with `To enable or disable this feature`, and that set my expectations that there would be info about enabling as well as disabling. It's not a big deal if you leave it as-is, but that was a question that came up for me when I read the paragraph, so maybe it's something for you to consider adding.

Reply:
Thank you! I decided to rework this section to make it (hopefully) less confusing and more minimalistic.